.circleci/config.yml

```diff
@@ -76,12 +76,7 @@ jobs:
       - restore_cache: *restore_cache
       - attach_workspace: { at: "." }
       - store_artifacts: { path: packages/docs/dist }
-      - run:
-          name: Submit Github comment with links to built artifacts
-          command: |
-            artifacts=$(curl -X GET "https://circleci.com/api/v2/project/github/palantir/documentalist/$CIRCLE_BUILD_NUM/artifacts" -H "Accept: application/json" -u "$CIRCLE_AUTH_TOKEN:")
-            echo $artifacts > ./scripts/artifacts.json
-            node ./scripts/circle-build-preview.js
+      - run: ./scripts/submit-preview-comment.sh
```

Comment:
is this PR supposed to hav ea comment on it?

Reply:
good Q, the preview comment is not working since we don't have the API token being injected right now but we do have docs built and saved as an artifact, which is most of the important part: https://app.circleci.com/pipelines/github/palantir/documentalist/389/workflows/8aba4d88-d87e-4dd5-a45f-ffdda3fad7a5/jobs/2679/artifacts